live-server.js

```diff
@@ -22,6 +22,15 @@ if (fs.existsSync(configPath)) {
 	if (opts.ignorePattern) opts.ignorePattern = new RegExp(opts.ignorePattern);
 }
 
+var dir = opts.root = process.argv[2] || "";
```

Comment:
I don't think moving this will work since looking at `process.argv[2]` here will return a random argument whereas in the old place the loop would have removed all recognized arguments.